src/main/java/de/tum/in/www1/artemis/service/connectors/localci/LocalCISharedBuildJobQueueService.java

```diff
@@ -185,12 +187,14 @@ public void saveFinishedBuildJob(LocalCIBuildJobQueueItem queueItem, BuildJobRes
             buildJob.setTriggeredByPushTo(queueItem.triggeredByPushTo());
             buildJob.setBuildJobResult(result);
             buildJob.setDockerImage(queueItem.dockerImage());
-
             buildJobRepository.save(buildJob);
         }
         catch (Exception e) {
             log.error("Could not save build job to database", e);
         }
+        addToRecentBuildJobs(new LocalCIBuildJobQueueItem(queueItem.id(), queueItem.name(), queueItem.buildAgentAddress(), queueItem.participationId(), queueItem.repositoryName(),
+                queueItem.repositoryType(), queueItem.commitHash(), queueItem.submissionDate(), queueItem.retryCount(), queueItem.buildStartDate(), buildCompletionDate,
+                queueItem.priority(), queueItem.courseId(), queueItem.triggeredByPushTo(), queueItem.dockerImage(), result.name()));
```

Comment:
That's a really long constructor... Can you simplify that to something like `new LocalCIBuildJobQueueItem(queueItem, buildCompletionDate, result.name())` which then copies the other values?